### PR TITLE
Submission README review

### PR DESCRIPTION
**Pull request by mpariente**
Find attached the current version of the submission README. 

Comments/review appreciated @popcornell @faroit @jonashaag @michelolzam 

Thanks !

### CODE REVIEW

File: egs/demask/README.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+[What it does](#what-it-does) •
+[How we built it](#how-we-built-it) •
+[Asteroid highlights](#asteroid-highlights) •
+[Challenges and Accomplishments](#challenges-we-ran-into) •
+[Acknowledgement](#acknowledgement)
+</div>
+
+## Inspiration
+
+#### Asteroid's inspiration
```
Comment (faroit) [inline egs/demask/README.md:12]: in this readme, demask inspiration should go first

Comment (faroit) [reply egs/demask/README.md:12]: i would rather put this to the bottom, naming it "about asteroid"

File: egs/demask/README.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+code with things that _just work_.
+
+While sharing research code is already a step in the right direction,
+sharing readable and reproducible code should be the standard.
+Asteroid aims at empowering developers and researchers with tools that makes this even easier.
+
+#### DeMask's inspiration
+
+We were having a fast call and one of us was in the train.
+We couldn't hear him well because he was wearing a mask.
```
Comment (faroit) [inline egs/demask/README.md:27]: him/her

File: egs/demask/README.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+We directly thought about building surgically masked speech enhancement model with
+Asteroid and we were on it !
```
Comment (faroit) [inline egs/demask/README.md:29]: i think it's a bit short. Maybe add a few more buzz words like covid, pandemic etc

File: egs/demask/README.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+## What it does
+
+#### About Asteroid
```
Comment (faroit) [inline egs/demask/README.md:33]: again, other order

File: egs/demask/README.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+You can check, [our landing page](https://asteroid-team.github.io/),
+[our repo](https://github.com/mpariente/asteroid),
+[our latest docs](https://mpariente.github.io/asteroid/)
+and [our model hub](https://zenodo.org/communities/asteroid-models).
+
+To try Asteroid, install it with `pip install asteroid` !
+
+#### About DeMask
+
+DeMask is a simple, yet effective, end-to-end muffled speech enhancement built with Asteroid.
```
Comment (faroit) [inline egs/demask/README.md:54]: i would remove "muffle" and just say "enhancement for speech when wearing a face mask"

File: egs/demask/README.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+It restores the frequency content which is distorted by the face mask,
+making the muffled speech sound cleaner.
```
Comment (faroit) [inline egs/demask/README.md:56]: here, muffled is ok

File: egs/demask/README.md (hunk 3 above)
Comment (mpariente) [reply egs/demask/README.md:29]: Feel free to make suggestions, please

File: egs/demask/README.md (hunk 5 above)
Comment (mpariente) [reply egs/demask/README.md:54]: Ok

File: egs/demask/README.md [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+- End to end separation to ASR with [ESPNet](https://github.com/espnet/espnet)
+and the list can go on..
+
+
+## More detail on how we improved Asteroid _during_ the hackathon period
+
+- Multiply by three the number of supported models
+- Improve model sharing by refactoring the publishing's base class.
+- End to end DSP
+- Inference engine with enhancement/separation CLI.
```
Comment (popcornell) [inline egs/demask/README.md:190]: maybe say Continuos Source Separation here also

Comment (mpariente) [reply egs/demask/README.md:190]: Yes, thanks

File: egs/demask/README.md [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+</p>
+
+## Challenges we ran into
+
+- In our first approach, we wanted to use the Compare dataset (classification with unpaired data),
+and use style transfer to perform enhancement but the amount of data was too small and the
+differences between mask vs. no-mask too subtle.
+When we got the impulse responses (IRs) from [Corey et al. 2020](https://arxiv.org/abs/2008.04521),
+none of our first ideas worked because the filters contained the IR of the microphone and the room, and the phase
+was noisy. Approximating the filters by hand to be linear-phase saved us !
```
Comment (popcornell) [inline egs/demask/README.md:147]: We then resorted to design ad-hoc FIR filters which directly approximate the frequency response of the masks in [Corey et al. 2020](https://arxiv.org/abs/2008.04521). These filters are dynamically generated at training time to augment the available data.

File: egs/demask/README.md [hunk 9]
```diff
@@ -0,0 +1,219 @@
...
+## What we learned
+Individually, we've learned to work as a team, set our goals, separate tasks and act fast.
+
+## What's next for Asteroid
+Pretty much everything is next.
+- A growing community of users and contributors.
+- A nice refactoring of the bash recipes into Python CLI.
+- A tighter integration with `torchaudio`.
+- Hopefully `TorchScript` support.
+- End to end separation to ASR with [ESPNet](https://github.com/espnet/espnet)
```
Comment (popcornell) [inline egs/demask/README.md:181]: going multi-channel ?

**Pull request closed without merging**